setools/infoflow.py

```diff
@@ -33,7 +34,7 @@ class InfoFlowAnalysis:
 
     """Information flow analysis."""
 
-    def __init__(self, policy, perm_map, min_weight=1, exclude=None):
+    def __init__(self, policy, perm_map, min_weight=1, exclude=None, conditionals=None):
```

Comment:
Same thing as above, should be `booleans` not `conditionals`. Also need an explanation of how the parameter is used in the comments below, along with the other Parameter descriptions.

The explanation should be that if it is `None`, all rules will be added to the analysis (default), otherwise it should be set to a dict with keys corresponding to Boolean names and values of T/F. Any unspecified Booleans will use the policy's default values.